httpreq/httpreq.go

```diff
@@ -17,21 +16,13 @@ const DefaultMaxIdleConnsPerHost = 100
 var defaultClient *http.Client
 
 func init() {
-	defaultTransportPointer, ok := http.DefaultTransport.(*http.Transport)
-	if !ok {
-		panic(fmt.Sprintf("defaultRoundTripper not an *http.Transport"))
-	}
-	defaultTransport := *defaultTransportPointer
-
-	// http.DefaultMaxIdleConnsPerHost = 2
-	defaultTransport.MaxIdleConnsPerHost = DefaultMaxIdleConnsPerHost
-
-	defaultClient = &http.Client{Transport: &defaultTransport}
+	defaultClient = &http.Client{}
```

Comment:
Setup `defaultClient.Transport` in `init` function.

Reply:
I updated this part